utils/elab_plugin.py

```diff
@@ -21,32 +21,35 @@ def setup(self, calling_class):
         """
         Setup, called before upload to iRods.
         """
-        config = calling_class.get_config('ELN')
+        token = calling_class.context.ibridges_configuration.config.get('eln_token', '')
+        #self.group = calling_class.context.ibridges_configuration.config.get('eln_group', '')
+        #self.experiment = calling_class.context.ibridges_configuration.config.get('eln_experiment', '')
+        #self.title = calling_class.context.ibridges_configuration.config.get('eln_title', '')
+        
+        in_var = input('Link data to ElabJournal experiment (Y/N, default N): ')
```

Comment:
If you add a strip() and a lower() (or upper), you need less options and are more certain of the input:
`
in_var = input('Link data to ElabJournal experiment (Y/N, default N): ').strip().lower()
if in_var in ['', 'n','no']:
`
or even
`
if input('Link data to ElabJournal experiment (Y/N, default N): ').strip().lower() in ['', 'n','no']:
`
Same applies to line 41.